recipes-bsp/tegra-binaries/tegra-bootfiles_35.3.1.bb

```diff
@@ -77,6 +80,25 @@ do_compile:append:tegra194() {
     truncate --size=393216 ${B}/nvdisp-init.bin
 }
 
+do_compile:append:tegra194() {
+    BL_DIR="${S}/bootloader"
+    # Create badpage.bin if it doesn't exist
+    if [ ! -f "${BL_DIR}/badpage.bin" ]; then
```

Comment:
Is there a need to reuse an existing file?  The result is the same: a 4k file filled with zeros via lines 88 or 92.

Reply:
This is the way NVIDIA implemented it in their flash tool. So I found it safer to just use their implementation, so the behavior between this layer and the NVIDIA flash script is limited